### PR TITLE
Added error404 page + improved homepage css (WIP)

### PR DESCRIPTION
**Pull request by realhanzalah**
if any collisions for homepage then use this version as its the improved one, still work in progress. should've done the error page as a separate branch to the homepage improvements but they're under the same branch now. homepage improvements ill put more under a new branch when I next work on it.

### CODE REVIEW

**Review by Aaron3455454 — APPROVED** [review]

**Comment by realhanzalah** [comment]
I've resolved conflicts now

File: view/error404.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>404 - Page Not Found</title>
+    <link rel="stylesheet" href="css/error404.css">
```
Comment (YourKalamity) [inline view/error404.php:7]: Please reference the full link for the CSS to ensure it’s compatible with the routing `/view/css/error404.css`

File: view/error404.php [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+</head>
+<body>
+    <div class="container">
+        <header>
+            <h1><strong>Error 404 - Page Not Found</strong></h1>
+    </header>
+        <p>Oops! The page you're looking for doesn't exist.</p>
+        <p>Return to <a href="home.php">home</a>.</p>
```
Comment (YourKalamity) [inline view/error404.php:15]: Change the href here to just home without any extensions

File: view/home.php [hunk 3]
```diff
@@ -8,7 +8,7 @@
         integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.2/font/bootstrap-icons.min.css">
     <link rel="stylesheet" type="text/css" href="view/css/home.css">
```
Comment (YourKalamity) [inline view/home.php:10]: Update the CSS linking here as well just like I stated in a previous comment

File: view/home.php [hunk 4]
```diff
@@ -18,21 +18,21 @@
 <body>
     <nav class="navbar navbar-expand-lg  fixed-top py-1">
         <div class="container-fluid">
-            <img src="view/images/Picture1.png" style="width: 50px; height: 50px;" alt="EvoTech logo">
-            <a class="navbar-brand me-auto" href="#">EvoTech</a>
+            <img src="images/Picture1.png" style="width: 50px; height: 50px;" alt="Evotech Logo">
```
Comment (YourKalamity) [inline view/home.php:21]: Another linking that needs to have view/ before images/ (view/images/…)

File: view/home.php [hunk 5]
```diff
@@ -18,21 +18,21 @@
...
                 <div class="offcanvas-header">
-                    <h5 class="offcanvas-title" id="offcanvasNavbarLabel">EvoTech</h5>
+                    <h5 class="offcanvas-title" id="offcanvasNavbarLabel">Evotech</h5>
                     <button type="button" class="btn-close" data-bs-dismiss="offcanvas" aria-label="Close"></button>
                 </div>
                 <div class="offcanvas-body">
                     <ul class="navbar-nav justify-content-center flex-grow-1 pe-3">
                         <li class="nav-item">
-                            <a class="nav-link active" aria-current="page" href="#">Home</a>
+                            <a class="nav-link active" aria-current="page" href="home.php">Home</a>
```
Comment (YourKalamity) [inline view/home.php:32]: This should just be home

File: view/home.php [hunk 6]
```diff
@@ -18,21 +18,21 @@
...
                         </li>
                         <li class="nav-item">
-                            <a class="nav-link" href="aboutus">About</a>
+                            <a class="nav-link" href="aboutus.php">About</a>
```
Comment (YourKalamity) [inline view/home.php:35]: This should just be aboutus

File: view/home.php [hunk 7]
```diff
@@ -65,17 +65,17 @@
...
+                    <h2>Evolve Your Gear</h2>
                     <p class="lead my-3">
-                        Shop the latest technology at evotech
+                    
                     </p>
-                    <button class="btn btn-primary btn-md"> Shop now</button>
+                    <button class="btn btn-primary btn-md">Shop Now</button>
                 </div>
-               <img class="img-fluid w-10" src="view/images/Picture1.png"alt="EvoTech"> 
+               <img class="img-fluid w-10" src="images/Picture1.png"alt="Evotech">
```
Comment (YourKalamity) [inline view/home.php:78]: This linking requires view before images

**Review by YourKalamity — CHANGES_REQUESTED** [review]

File: view/home.php (hunk 6 above)
Comment (YourKalamity) [reply view/home.php:35]: Without the php extension

**Review by YourKalamity — APPROVED** [review]

**Pull request merged**